packages/app-page-builder/src/render/components/Element.tsx

```diff
@@ -27,7 +27,20 @@ const Element = (props: ElementProps) => {
         return null;
     }
 
-    return <>{plugin.render({ theme, element })}</>;
+    let renderedPlugin = null
```

Comment:
Since this is an exact copy, maybe we could extract this whole tool into a separate utility. I'm thinking maybe `app-page-builder/src/utils/renderElementPlugin.tsx`? Then import and use it in both editor and render Element. Let's do that and see how it looks.

Reply:
Here, I've done so